dali/core/CMakeLists.txt

```diff
@@ -31,10 +31,11 @@ endforeach(incl_dir)
 separate_arguments(DEFAULT_COMPILER_INCLUDE UNIX_COMMAND  "${DEFAULT_COMPILER_INCLUDE}")
 
 if (NOT LINK_DRIVER)
+  find_package (Python COMPONENTS Interpreter)
   set(CUDA_GENERATED_STUB "${CMAKE_CURRENT_BINARY_DIR}/dynlink_cuda_gen.cc")
   add_custom_command(
       OUTPUT ${CUDA_GENERATED_STUB}
-      COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/../../tools/stub_generator/stub_codegen.py --unique_prefix=Cuda --
+      COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/../../tools/stub_generator/stub_codegen.py --unique_prefix=Cuda --
```

Comment:
@JanuszL Are we sure it won't break anything in our build system?

Reply:
It executes python that is visible in the shell. I guess `Python_EXECUTABLE` is the python that CMake can find, and probably available in the shell.
I think it should be fine.

Reply:
In some systems, there is no python but python3...